www/AuthenticationResult.js

```diff
@@ -17,7 +17,16 @@ function AuthenticationResult(authResult) {
     this.statusCode = authResult.statusCode;
     this.tenantId = authResult.tenantId;
 
-    this.userInfo = authResult.idToken ? UserInfo.fromJWT(authResult.idToken) : null;
+    var jwtToken = authResult.idToken || authResult.accessToken;
+    this.userInfo = null;
+
+    if (jwtToken) {
+        this.userInfo = UserInfo.fromJWT(jwtToken);
+    }
+
+    if (!this.userInfo) {
+        this.userInfo = new UserInfo(authResult.userInfo);
```

Comment:
@daserge, yes, it's verified [there](https://github.com/AzureAD/azure-activedirectory-library-for-cordova/blob/master/www/UserInfo.js#L12)